virtcontainers/sandbox.go

```diff
@@ -1666,3 +1652,27 @@ func (s *Sandbox) updateResources() error {
 	}
 	return nil
 }
+
+func calculateSandboxMemory(containers []ContainerConfig) int64 {
```

Comment:
self review: probably can convert to a sandbox method and do not receive containers as function parameter. 
@devimc @jodh-intel any preference here?

Reply:
or you can move them to utils

Reply:
Yeah, I'd either make it a sandbox method, or just pass it the sandbox and put it in utils as @devimc suggests.

/cc @sboeuf, @bergwolf.

Reply:
I would keep that in `sandbox.go` as a sandbox method. This makes more sense IMO as I don't see this function being reused by anything else since it is very sandbox specific. The code we put in utils should be very generic so that it could be reused by any piece of code.
And I think `calculateSandboxCPUs()` is actually the perfect example where it is a sandbox method, that relies onto some very generic `utils.CalculateMilliCPUs()` function calls.

Reply:
thank you all, moved as sandbox method.